Index: server/src/Logica/Listas/ListaDouble.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package Logica.Listas;\r\n\r\npublic class ListaDouble {\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/server/src/Logica/Listas/ListaDouble.java b/server/src/Logica/Listas/ListaDouble.java
--- a/server/src/Logica/Listas/ListaDouble.java	(revision 8af66a8a27ec6dba41ef9000b982442f15e481f0)
+++ b/server/src/Logica/Listas/ListaDouble.java	(date 1710527614986)
@@ -1,4 +1,3 @@
 package Logica.Listas;
-
 public class ListaDouble {
 }
